<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modeChooser</class>
 <widget class="QDialog" name="modeChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>184</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="title">
        <string>Choose session mode:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <widget class="QRadioButton" name="ButtonBoxes">
            <property name="text">
             <string>Boxes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="boxesInfoButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../Resources/sanki.qrc">
              <normaloff>:/icons/info.svg</normaloff>:/icons/info.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <widget class="QRadioButton" name="ButtonCRandom">
            <property name="text">
             <string>Completly randomised</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CRInfoButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../Resources/sanki.qrc">
              <normaloff>:/icons/info.svg</normaloff>:/icons/info.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="ButtonRandomNR">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Randomised - No repeating</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="ButtonConfirm">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/sanki.qrc"/>
 </resources>
 <connections/>
</ui>
